src/main/java/com/limechain/network/protocol/warp/dto/BlockHeader.java

```diff
@@ -38,4 +38,9 @@ public Hash256 getHash() {
         byte[] scaleEncoded = ScaleUtils.Encode.encode(BlockHeaderScaleWriter.getInstance(), this);
         return new Hash256(HashUtils.hashWithBlake2b(scaleEncoded));
     }
+
+    public byte[] getHashBytes() {
```

Comment:
This method is almost the same as the one above, `getHash()`. We can merge them.